src/build.rs

```diff
@@ -0,0 +1,518 @@
+use std::collections::HashSet;
+
+use crate::{data::*, prelude::*, HolidayMap, Result, Year};
+
+/// Generate holiday map for the specified countries and years.
+pub fn build(
```

Comment:
Can you merge src/builder.rs into src/build.rs?

Reply:
I wouldn't merge these two into a singe file, because build.rs is generated, builder.rs is normal Rust code